browser-test/src/support/applicant_questions.ts

```diff
@@ -522,11 +522,13 @@ export class ApplicantQuestions {
     await this.page.click(`:nth-match(:text("Remove entity"), ${entityIndex})`)
   }
 
-  async downloadSingleQuestionFromReviewPage() {
+  async downloadSingleQuestionFromReviewPage(northStarEnabled = false) {
     // Assert that we're on the review page.
-    expect(await this.page.innerText('h2')).toContain(
-      'Program application summary',
-    )
+    if (northStarEnabled) {
+      await expect(this.page.getByText('Review and submit')).toBeVisible()
+    } else {
+      expect(this.page.innerText('h2')).toContain('Program application summary')
```

Comment:
```suggestion
      expect(await this.page.innerText('h2')).toContain('Program application summary')
```

Reply:
Your commit has expect and await swapped

Reply:
I thought `await expect(...)` is preferred?

https://playwright.dev/docs/best-practices#use-web-first-assertions

Reply:
It is, but that entire line would need to be rewritten to use newer methods that support it. Given it will eventually go away when northstar goes live I wouldn't bother.

Reply:
Oh! you did bother. Nice job! :)